doc/sidewalk_known_issues.rst

```diff
@@ -5,10 +5,23 @@ Known issues
 
 See the list of known issues that are valid for the current state of development.
 
-v1.14.4
--------
+v1.14.5-dev1
```

Comment:
version of tag to be checked, before was discussed 1.14.4 dev1 as next release is going to be 1.14.5 dev1

Reply:
As discussed offline - v1.14.5 dev1